### PR TITLE
Feature/nec 20/add lithuanian language

### PR DESCRIPTION
**Pull request by AndreyNegrov**
add new language (lt-LT)

### CODE REVIEW

**Comment by tikhanovichA** [comment]
Is it ok, that I see lines without translates?

**Comment by AndreyNegrov** [comment]
> Is it ok, that I see lines without translates?

it's normal

File: locales/lt-LT/messages.po [hunk 1]
```diff
@@ -0,0 +1,1568 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/messages.po]: This is not correct.

Comment (AndreyNegrov) [reply locales/lt-LT/messages.po]: done

File: locales/lt-LT/qtiCat.rdf.po [hunk 2]
```diff
@@ -1,14 +1,18 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: TAO 3.4.0-sprint112\n"
-"PO-Revision-Date: 2019-10-09T09:03:03\n"
-"Last-Translator: TAO Translation Team <translation@tao.lu>\n"
-"MIME-Version: 1.0\n"
+"Project-Id-Version: PACKAGE VERSION\n"
```
[the remark this answers is not in the document]
Comment (AndreyNegrov) [reply locales/lt-LT/qtiCat.rdf.po]: done

**Pull request merged**